Mage/src/main/java/mage/abilities/effects/common/ExileTargetCardCopyAndCastEffect.java

```diff
@@ -0,0 +1,59 @@
+package mage.abilities.effects.common;
+
+import mage.ApprovingObject;
+import mage.abilities.Ability;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.OneShotEffect;
+import mage.cards.Card;
+import mage.constants.Outcome;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.players.Player;
+
+public class ExileTargetCardCopyAndCastEffect extends OneShotEffect {
```

Comment:
Currently this effect only casts without paying mana cost. I think it could easily be expanded in scope to also work for effects that allow casting a copy still paying costs? e.g. [[Nashi, Moon's Legacy]] or [[Demilich]]
(Otherwise, maybe rename to `ExileTargetCardCopyAndCastForFreeEffect` for clarity)

Can also use in:
[[Arcane Proxy]]
[[Founding the Third Path]]
[[Narset, Enlightened Exile]]
[[Psionic Ritual]]

Reply:
Oh yeah, let me do that. I didn't realise that there were such cards.

Reply:
Also can be used for Mizzix's Mastery